test/dateandtime.py

```diff
@@ -84,34 +86,34 @@ def suite():
     """
 #    # date_parser
```

Comment:
just remove all the commented prints

Reply:
or better yet, remove everyting after `def suite` above

Reply:
done